Assets/MRTK/SDK/Features/UX/Scripts/HandCoach/MoveToTarget.cs

```diff
@@ -15,7 +15,7 @@ public class MoveToTarget : MonoBehaviour
         /// <summary>
         /// Object to track.
         /// </summary>
-        public GameObject TrackingObject
+        private GameObject TrackingObject
```

Comment:
Changing these from `public` to `private` is a breaking API change (which we try to avoid as much as possible), and I'm also not sure why they need to be changed. These are all helper scripts that could be called from anybody's application, even if MRTK doesn't call them directly.

Reply:
Ah my bad, i didn't know that, thanks for the review, I've reverted it.